pkgs/swift2objc/lib/src/parser/parsers/declaration_parsers/parse_function_declaration.dart

```diff
@@ -142,6 +143,13 @@ ParsedFunctionInfo parseFunctionInfo(
   );
 }
 
+bool _parseFunctionIsMutating(Json methodSymbolJson) {
```

Comment:
It'd be cleaner to do this in `parseFunctionInfo`, and add a flag to `ParsedFunctionInfo`, even though not all kinds of function support `mutating`.

That will also mean you can test it in `test/unit/parse_function_info_test.dart`.